src/analysis/calculus/times_cont_diff.lean

```diff
@@ -1880,7 +1880,11 @@ begin
     exact (h i).zero_eq x hx },
   { intros m hm x hx,
     have := has_fderiv_within_at_pi.2 (λ i, (h i).fderiv_within m hm x hx),
-    convert (L m).has_fderiv_at.comp_has_fderiv_within_at x this },
+    convert (linear_isometry_equiv.has_fderiv_at _).comp_has_fderiv_within_at x this,
+    swap 3,
+    exact L m,
+    refl,
+    refl },
```

Comment:
I would leave out the swap, and add `{ .. }` around the 1-line subproofs.

Reply:
@sgouezel do you have any idea why the previous proof breaks here?

Reply:
I think that I tried leaving out the swap, and that it made the refl's fail.

Reply:
I've been investigating this over the last couple of hours and might be on to something...will update soon hopefully.

Reply:
I see. In that case, please add a comment
```suggestion
    swap 3, -- without this swap, the `refl`s fail
    { exact L m },
    { refl },
    { refl } },
```